libraries/gl/src/gl/Config.cpp

```diff
@@ -124,3 +132,14 @@ void gl::setSwapInterval(int interval) {
     Q_UNUSED(interval);
 #endif
 }
+
+bool gl::queryCurrentRendererIntegerMESA(int attr, unsigned int *value) {
+    #if defined(Q_OS_LINUX)
```

Comment:
do you need this ifdef, since you already check if the function pointer below?

Reply:
The variable that holds the pointer is declared in an #ifdef and may not exist